va/va_cp_h264.h

```diff
@@ -0,0 +1,481 @@
+    uint8_t frame_cropping_flag;
+    /** \brief Same as the H.264 bitstream syntax element. */
+    uint32_t frame_crop_left_offset;
+    /** \brief Same as the H.264 bitstream syntax element. */
+    uint32_t frame_crop_right_offset;
+    /** \brief Same as the H.264 bitstream syntax element. */
+    uint32_t frame_crop_top_offset;
+    /** \brief Same as the H.264 bitstream syntax element. */
+    uint32_t frame_crop_bottom_offset;
+    /**@}*/
+
+    /** @name VUI parameters (optional) */
+    /**@{*/
+    /** \brief Same as the H.264 bitstream syntax element. */
+    uint8_t   vui_parameters_present_flag;
+    union {
+        struct {
+            /** \brief Same as the H.264 bitstream syntax element. */
+            uint32_t aspect_ratio_info_present_flag         : 1;
+            /** \brief Same as the H.264 bitstream syntax element. */
+            uint32_t video_signal_type_present_flag         : 1;
+            /** \brief Same as the H.264 bitstream syntax element. */
+            uint32_t colour_description_present_flag        : 1;
+            /** \brief Same as the H.264 bitstream syntax element. */
+            uint32_t timing_info_present_flag               : 1;
+            /** \brief Same as the H.264 bitstream syntax element. */
+            uint32_t fixed_frame_rate_flag                  : 1;
+            /** \brief Same as the H.264 bitstream syntax element. */
+            uint32_t nal_hrd_parameters_present_flag        : 1;
+            /** \brief Same as the H.264 bitstream syntax element. */
+            uint32_t vcl_hrd_parameters_present_flag        : 1;
+            /** \brief Same as the H.264 bitstream syntax element. */
+            uint32_t low_delay_hrd_flag                     : 1;
+            /** \brief Same as the H.264 bitstream syntax element. */
+            uint32_t pic_struct_present_flag                : 1;
+            /** \brief Same as the H.264 bitstream syntax element. */
+            uint32_t bitstream_restriction_flag             : 1;            
+            /** \brief Reserved for future use, must be zero */
+            uint32_t reserved                               : 22;
+        } bits;
+        uint32_t value;
+    } vui_fields;
+    /** \brief Same as the H.264 bitstream syntax element. */
+    uint8_t aspect_ratio_idc;
+    /** \brief Same as the H.264 bitstream syntax element. */
+    uint32_t sar_width;
+    /** \brief Same as the H.264 bitstream syntax element. */
+    uint32_t sar_height;
+    /** \brief Same as the H.264 bitstream syntax element. */
+    uint8_t video_format;
+    /** \brief Same as the H.264 bitstream syntax element. */
+    uint8_t colour_primaries;
+    /** \brief Same as the H.264 bitstream syntax element. */
+    uint8_t transfer_characteristics;
+    /** \brief Same as the H.264 bitstream syntax element. */
+    uint8_t matrix_coefficients;
+    /** \brief Same as the H.264 bitstream syntax element. */
```

Comment:
Chroma sample location information is for interlaced 4:2:0 format, right? Since most of premium contents are progressive(We didn't see the interlaced premium contents), can chroma sample location be excluded from vui?

Reply:
No, it's used for all 4:2:0 cases.  (When the stream is progressive the top and bottom field values are required to be equal.)

Reply:
This struct is removed due to SPS is returned as bitstream.